test/arrays.jl

```diff
@@ -17,6 +17,23 @@ using Images, IndirectArrays, Test
     for (a,t) in zip(A, target)
         @test a === t
     end
+
+    intensity1 = [0.1 0.3 0.6; 0.2 0.4 0.1]
+   labels1 = IndirectArray([1 2 3; 2 1 3], [RGB(1,0,0), RGB(0,0,1),RGB(0,1,0)])
+   A1 = ColorizedArray(intensity1, labels1)
+   target1 = intensity1 .* labels1
+   @test eltype(A1) == RGB{Float64}
+   @test size(A1) == (2,3)
+   @test axes(A1) == (Base.OneTo(2), Base.OneTo(3))
```

Comment:
```suggestion
   @test axes(A1) == (1:2, 1:3)
```
unless you think it's important to test type-equality, in which case you need to use `===` rather than `==` (and I don't see why that would be important).